internal/runners/platforms/add.go

```diff
@@ -98,7 +98,7 @@ func (a *Add) Run(params AddRunParams) (rerr error) {
 
 	// Prepare updated buildscript
 	script := oldCommit.BuildScript()
-	script.SetAtTime(ts)
+	script.SetAtTime(ts, false)
```

Comment:
```suggestion
	script.SetAtTime(ts, true)
```

We're making a change, which by definition means we're bumping the atTime (unless the current atTime is higher than the latest, but that's handled by the Expand function).